src/node.rs

```diff
@@ -112,10 +119,27 @@ impl Node {
 
         Some((class_name, class_version))
     }
+
+    pub fn cmp_by_id(&self, other: &Node) -> cmp::Ordering {
```

Comment:
We probably don't any more. When the nodes were in a Vec I used it as an ordering function passed to sort. Now that we're using BTreeSet we don't need it any more.